### DISCUSSION TITLE
Create a default VR Config

### DISCUSSION THREAD

**Discussion by cajunjosh** [magknight/787aviatorsTracker#711] [Completed things] [answered]
Create a default B789_crconfig.txt VR Config file that Skunkscraft won't delete during each update.  I've included mine below as a starting point.  

A
1100
VRCONFIG

BEGIN_TELEPORT_HOTSPOT SITTING Pilot View
   AABB -0.8608 0.7479 -25.7336 -0.2608 0.9479 -25.1336
   PRESET_XYZ -0.5608 1.3979 -25.4336
   PRESET_PSI 0.0000
   PRESET_THE 0.0000
   PRESET_PHI 0.0000
END_TELEPORT_HOTSPOT

################################
#      TELEPORT HOTSPOTS       
################################

BEGIN_TELEPORT_HOTSPOT SITTING Pilot View
   AABB -0.8608 0.7479 -25.7336 -0.2608 0.9479 -25.1336
   PRESET_XYZ -0.5608 1.3979 -25.4336
   PRESET_PSI 0.0000
   PRESET_THE 0.0000
   PRESET_PHI 0.0000
END_TELEPORT_HOTSPOT

**Comment by aeroniemi** [comment, marked as answer]
I've added that hotspot as part of the 1.7.24 - we probably won't make a extensive vrconfig ourselves as we don't support VR officially, but if you or anyone else wants to submit a more detailed one we'd be happy to add it
